src/actor.ts

```diff
@@ -24,7 +24,9 @@ export async function createActor(
     .getPeerListenAddresses()
     .then(addresses => addresses[0]);
 
-  const comitClient = new ComitClient(bitcoinWallet, ethereumWallet, cnd);
+  const comitClient = new ComitClient(cnd);
+  comitClient.ethereumWallet = ethereumWallet;
+  comitClient.bitcoinWallet = bitcoinWallet;
```

Comment:
Instead of exposing these fields publicly you could apply the _builder pattern_. This would allow for something like this: 

```
const comitClient = new ComitClient(cnd)
    .withEthereumWallet(ethereumWallet)
    .withBitcoinWallet(bitcoinWallet)
    ....;
```

Let me know what you think and I can provide a small PR once your changes are in.

Reply:
Please hold, I am finishing off few things. I'll ping you.